app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -47,10 +47,14 @@
 				v-model:headers="headers"
 				:class="{ 'no-last-border': totalItemCount <= 10 }"
 				:loading="loading"
+				:show-manual-sort="allowDrag"
```

Comment:
Can we disable the manual sort when the total amount of items is bigger than the configured page size? (Similarly to how the regular list-style works 🙂 ) That should avoid a bit of confusion in the expectation that you could otherwise sort across pages 👍🏻

Reply:
This is already handled by the allowDrag computed so no need for extra logic